conf/vagrant/provisioning/roles/php/tasks/main.yml

```diff
@@ -1,12 +1,12 @@
 ---
 - name: PHP | Adjust php.ini memory_limit
   become: true
-  lineinfile: dest={{ item }}
+  lineinfile: dest={{ item.file }}
               regexp='memory_limit'
-              line='memory_limit = {{ php_ini_memory_limit }}'
+              line='memory_limit = {{ item.value }}'
   with_items:
-    - /etc/php/5.6/apache2/php.ini
-    - /etc/php/5.6/cli/php.ini
+    - ( file: '/etc/php/5.6/apache2/php.ini', value: "{{ php_ini_memory_limit }}" }
```

Comment:
Is there a syntax error here? This line (and the next) start with a parenthesis `(` and end with a bracket `}`

Reply:
I'd say so.  Yikes!